AppKit/CPSegmentedControl.j

```diff
@@ -677,63 +707,39 @@ CPSegmentSwitchTrackingMomentary = 2;
 {
 }
 
-- (void)tileWithChangedSegment:(unsigned)aSegment
```

Comment:
This method is public right now. It's a method which not exist in the AppKit or in the UIKit.

Should we pass it in private even if it's was public for a while ?